crusher/compilers.yaml

```diff
@@ -29,28 +29,31 @@ compilers:
     - /opt/cray/pe/gcc-libs
     - /opt/cray/gcc-libs
 - compiler:
-    spec: cce@14.0.0
+    spec: cce@14.0.2
     paths:
-      cc: /opt/cray/pe/craype/2.7.13/bin/cc
-      cxx: /opt/cray/pe/craype/2.7.13/bin/CC
-      f77: /opt/cray/pe/craype/2.7.13/bin/ftn
-      fc: /opt/cray/pe/craype/2.7.13/bin/ftn
+      cc: cc
+      cxx: CC
+      f77: ftn
+      fc: ftn
     flags: {}
     operating_system: sles15
     target: any
     modules:
     - PrgEnv-cray/8.3.3
-    - cce/14.0.0
+    - cce/14.0.2
+    - xpmem
```

Comment:
just lamenting, but it is crazy we have to load this...

Reply:
I agree.